src/prefect/environments/storage/docker.py

```diff
@@ -174,10 +245,12 @@ def create_dockerfile_object_from_base_image(
             print('Beginning health check...')
             from prefect.utilities.environments import from_file
 
-            local_env = from_file('/root/.prefect/flow_env.prefect')
+            local_env = from_file('{flow_file_path}')
```

Comment:
this appears to be missing a `.format(flow_file_path=self.flow_file_path)`

Reply:
No it isn't :) look a couple lines below

Reply:
Ahhhhh gotcha. 👍